sponsors/models.py

```diff
@@ -513,6 +530,13 @@ def new_copy(cls, benefit, **kwargs):
             **kwargs,
         )
 
+        # generate benefit features from benefit features configurations
+        for feature_config in benefit.features_config.all():
+            feature = feature_config.get_benefit_feature(sponsor_benefit=sponsor_benefit)
```

Comment:
I really appreciate that this is a method that can be defined! Will be super helpful in setting up things like "Deadlines" which may be dependent on the date of approval!